File: objectModel/Python/cdm/persistence/cdmfolder/types/projections/operation_exclude_attributes.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from typing import List

from cdm.persistence.cdmfolder.types.projections.operation_base import OperationBase


class OperationExcludeAttributes(OperationBase):
    """OperationExcludeAttributes class"""

    def __init__(self):
        super().__init__()

        self.excludeAttributes = None  # type: List[str]

File: objectModel/Java/objectmodel/src/main/java/com/microsoft/commondatamodel/objectmodel/persistence/cdmfolder/types/projections/OperationExcludeAttributes.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

package com.microsoft.commondatamodel.objectmodel.persistence.cdmfolder.types.projections;

import java.util.List;

/**
 * OperationExcludeAttributes class
 */
public class OperationExcludeAttributes extends OperationBase {
    private List<String> excludeAttributes;

    public List<String> getExcludeAttributes() {
        return excludeAttributes;
    }

    public void setExcludeAttributes(final List<String> excludeAttributes) {
        this.excludeAttributes = excludeAttributes;
    }
}
